network_definitions/bandwidth_per_network_service.yaml

```diff
@@ -0,0 +1,45 @@
+#flp_confgen
```

Comment:
taking into account you have [bandwidth_per_network_service_per_namespace.yaml](https://github.com/netobserv/network-observability-operator/pull/126/files#diff-42965142281d83341fae1005c26d37a1d4391498ea79607c1af72fe9f43b935d) --- this metric is not really needed. Can you remove it????

Reply:
Eventually, yes. In the meantime, we have no metrics being produced for these. At one point, using the network simulator, I got metrics per network_service but not per network_service_per_namespace, since the network simulator did not simulate connections from inside the cluster (hence no mapping to namespace). Once we have metrics reporting working properly, we should again visit the list of metrics.